docs/modules/globby.md

```diff
@@ -0,0 +1,46 @@
+# globby
+
+`globby` is a globbing library that presents [some behaviors](https://github.com/sindresorhus/globby#features)
+no other libraries have, mainly the handling of negated patterns and directory expansion. It can be replaced
+with lighter and faster alternatives.
+
+# Alternatives
+
+## tinyglobby
+
+`tinyglobby` can be used as an alternative in most cases.
```

Comment:
i would reword this as something like:

> `tinyglobby` is a smaller alternative. It does not have the same options as globby but often can be a drop-in replacement.

Reply:
i feel like that would be misleading as it does have the same options and api, its just that it doesn't cover all of them yet

Reply:
> It doesn't include all `globby` options though.

is the part i feel could be worded better

we're more concerned with the common use case than whatever edge case options globby might support. if tinyglobby is a faster alternative, we should make that clear. if it is lighter, that too. ultimately these libs are used mostly just to glob a simple pattern and we don't need to focus on what options are available